### ISSUE TITLE
Deprecate R4-PA-C23

### ISSUE THREAD

**Issue by naved001** [CCI-MOC/ops-issues#1069]
This currently hosts the ESI dev servers (kumo) so we'll need to arrange hardware for that.

https://github.com/CCI-MOC/esi/issues/418

The cisco nexus switch in this rack brings us the BU public network and the CSAIL 3803 network (MaaS network), so anything on that network will be down while we move things around.

**Comment by naved001** [comment]
@hakasapl as part of this, is it possible to move the drives from slot-12 dell in the c23 blades to MOC-R4PAC21U21-S3? They are 2 drives in RAID1 (hopefully the raid controller can rebuild the raid volume without loss of information). 
@tzumainn is currently copying the data out of it, but it would be nice if we could move the drives and boot the other system from it.

**Comment by naved001** [comment]
Data was backed up. Once we move the BU network, this rack can be deprecated.

**Comment by hakasapl** [comment]
I believe we are no longer using the BU network and this can be decommissioned. Can you confirm @naved001?

**Comment by naved001** [comment]
Correct, there's nothing live on the BU network. This also means that the cisco switch in that rack could be used for your dev purposes (it's running NXOS)

**Comment by naved001** [comment]
@hakasapl can we say this rack is deprecated or is there anything else that we need to do?

**Comment by hakasapl** [comment]
@naved001 There is still equipment in there that is unplugged, but that can stay until we actually replace it. BU public net comes in there as well but we're not using. So yes I think this is done
